src/components/map/Map.tsx

```diff
@@ -157,141 +179,256 @@ export const Map = ({
     };
   }
 
+  const mapLocations = locations.map((location, index) => ({
+    ...location,
+    type: 'Feature',
+    properties: { cluster: false, id: location.id, index },
+    geometry: {
+      type: 'Point',
+      coordinates: [location.position.longitude, location.position.latitude]
+    }
+  }));
+
+  useEffect(() => {
+    if (clusteringEnabled) {
+      const index = new Supercluster({
+        radius: clusterDistance,
+        maxZoom: MAX_ZOOM_LEVEL
+      });
+      const bounds = [2, 46, 18, 56];
```

Comment:
please add the comment about the bound numbers, which they are representing, directly in the code here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1154/commits/800969674da23ee9825df39f73ddb943008dbe34